src/tools/pbs_upgrade_job.c

```diff
@@ -196,7 +196,7 @@ check_job_file(int fd)
 	/* Save our current position so we can comeback to it */
 	pos_saved = lseek(fd, 0, SEEK_CUR);
 
-	/*---------- For PBSPro >=13.x or <=18.x versions jobfix structure ---------- */
+	/*---------- For PBS >=13.x or <=18.x versions jobfix structure ---------- */
```

Comment:
This could probably be left as PBSPro because that was the name of the product for versions 13-18. Only noting it here, but there are additional instances.

Reply:
@mike0042 It was decided to change all "PBSPro", "PBS Pro", "PBS Professional" in code to either just "PBS"

But let's see what @scc138 and @subhasisb guys think?

Reply:
Seems fine to just use PBS here -- otherwise, when you later grep for PBS Pro, you'll have to find all the places that you decided it was "OK", and that is error prone.